src/main/groovy/edu/oregonstate/mist/tdwrapper/db/TDServicesStaticJsonDAO.groovy

```diff
@@ -0,0 +1,66 @@
+package edu.oregonstate.mist.tdwrapper.db
+
+import edu.oregonstate.mist.api.jsonapi.ResourceObject
+import groovy.json.JsonSlurper
+
+class TDServicesStaticJsonDAO {
+
+    private File servicesFile
+
+    TDServicesStaticJsonDAO(String sourceJsonFile) {
+        servicesFile = new File(sourceJsonFile)
+    }
+
+    /**
+     * Get all services
+     * @return
+     */
+    public List<ResourceObject> getTDServices() {
+        def servicesRaw = getJsonRaw()
+
+        List<ResourceObject> services = []
+
+        servicesRaw.each { id, service ->
+            services.add(getResourceObject(service))
+        }
+
+        services
+    }
+
+    /**
+     * Get service by ID
+     * @param id
+     * @return
+     */
+    public ResourceObject getTDServiceByID(String id) {
+        def servicesRaw = getJsonRaw()
```

Comment:
do we really want to read the file on every single request?

Reply:
Yes, the services.json file will be updated every 30 minutes.